pallets/crowdloan-claim/src/mock.rs

```diff
@@ -63,6 +63,26 @@ impl WeightInfo for MockWeightInfo {
     fn claim_reward() -> Weight {
         0 as Weight
     }
+
+    fn set_lease_start() -> u64 {
+        0 as Weight
```

Comment:
so benchmarking is done in a different PR ?

Reply:
Yes. Once this is through I will add a PR for this pallet in parachain, where I integrate the benchmark. Wanted to close this first, finally...